src/pages/workspaces/List.js

```diff
@@ -249,27 +251,16 @@ export const WorkspaceList = () => {
           },
           size: { basis: 120, grow: 1, shrink: 0 }
         }, {
-          headerRenderer: () => div({ className: 'sr-only' }, ['Actions']),
+          headerRenderer: () => div({ className: 'sr-only' }),
```

Comment:
Same note here about the column needing a name so it says something meaningful when you navigate through the column headers with the screen reader.